extensions/wikia/AdEngine/js/video/player/ooyala/ooyalaTracker.js

```diff
@@ -0,0 +1,54 @@
+/*global define*/
+define('ext.wikia.adEngine.video.player.ooyala.ooyalaTracker', [
+	'ext.wikia.adEngine.video.player.playerTracker',
+	'wikia.window'
+], function (playerTracker, win) {
+	'use strict';
+	var playerName = 'ooyala',
+		trackingEventsMap = {
+			'adsClickthroughOpened': 'clicked',
+			'adsError': 'error',
+			'adsPlayed': 'completed',
+			'willPlayAds': 'loaded',
+			'playerCreated': 'ready',
+			'willPauseAds': 'paused',
+			'willResumeAds': 'resumed',
+			'willPlaySingleAd': 'started',
+
+			'willPlay': 'content_started',
+			'played': 'content_completed'
```

Comment:
what do you think about using contants provided by Ooyala?
eg. `OO.EVENTS.PLAYED`